src/components/Profile/TabPanel.js

```diff
@@ -0,0 +1,91 @@
+import React, {useEffect, useState} from 'react';
+import PropTypes from 'prop-types';
+import AppBar from '@material-ui/core/AppBar';
+import Tabs from '@material-ui/core/Tabs';
+import Tab from '@material-ui/core/Tab';
+import Box from '@material-ui/core/Box';
+import Item from 'components/Item/Item'
+
+import * as ApiService from 'ApiService/ApiService'
+import Table from 'components/Profile/Table'
+import SubTab from 'components/Profile/SubTab'
+import ProfileCard from 'components/Profile/ProfileCard';
+
+function TabPanel(props) {
+    const {
+        children,
+        value,
+        index,
+        ...other
+    } = props;
+
+    return (
+        <div
+            role="tabpanel"
+            hidden={value !== index}
+            id={`simple-tabpanel-${index}`}
+            aria-labelledby={`simple-tab-${index}`}
+            {...other}>
+            {value === index && (
+                <Box p={3}>
+                    <div>{children}</div>
+                </Box>
+            )}
+        </div>
+    );
+}
+
+TabPanel.propTypes = {
+    children: PropTypes.node,
+    index: PropTypes.any.isRequired,
+    value: PropTypes.any.isRequired
+};
+
+function a11yProps(index) {
+    return {id: `simple-tab-${index}`, 'aria-controls': `simple-tabpanel-${index}`};
+}
+
+function SimpleTabs() {
+    const [value,
+        setValue] = React.useState(0);
+
+    const handleChange = (event, newValue) => {
+        setValue(newValue);
+    };
+    
+    function createData(Name, Price, Label) {
+        return { Name, Price, Label };
+      }
+      
+      const rows = [
+        createData('Frozen yoghurt', 159, "Top rated"),
```

Comment:
Why hardcoding?

Reply:
This is temporary until functionality is implemented.